src/activatable/activatable.ts

```diff
@@ -27,20 +29,22 @@ export interface IActivatable extends IMinimalActivatable {
 }
 
 export abstract class Activatable implements IActivatable {
+  @observable
   private _state: 'deactivated' | 'isActivating' | 'activated' | 'isDeactivating' = 'deactivated';
```

Comment:
`type ActivatableState = 'deactivated' | 'isActivating' | 'activated' | 'isDeactivating'`

```suggestion
  private _state: ActivatableState = 'deactivated';
```